packages/openapi-spec-builder/README.md

```diff
@@ -37,31 +37,46 @@ const spec = anOpenApiSpec()
   .build();
 
 // which is equivalent to the following longer form
-
 const spec = new OpenApiSpecBuilder()
   .withOperation('get', '/hello', {
     'x-operation-name': 'greet',
     responses: {
-      '200': {type: 'string'},
+      '200': {
+        description: 'The string result.',
+        content: {
+          'text/plain': {
+            schema: {
+              type: 'string',
+            },
+          },
+        },
+      },
     },
   })
   .build();
 
 // the spec
-
-const spec = {
-  basePath: '/',
-  swagger: '2.0',
+const spec3 = {
```

Comment:
nitpick: `spec` is good I think :)

Reply:
you're right. We were copying over from the test file.  Switching back to `spec`.